pulp_smash/tests/rpm/cli/test_upload.py

```diff
@@ -14,7 +21,115 @@ def setUpModule():  # pylint:disable=invalid-name
     utils.pulp_admin_login(config.get_config())
 
 
-class UploadDrpmTestCase(unittest.TestCase):
+class _BaseCLIDownloadTestCase(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        cls.cfg = config.get_config()
+        cls.client = cli.Client(cls.cfg)
+
+    def temp_download(self, file_url, filename='tmp.rpm'):
+        """Download a file from the given url to a temp directory.
+
+        Args:
+            file_url: String. The URL of the file to download. Required
+            filename: String. Assigns a name to the file downloaded. Optional.
```

Comment:
Sphinx isn't configured to parse Google/napoleon-style docstrings. Please use the format that Sphinx expects:

```
:param file_url: The URL of the file to download.
```

It's not necessary to write that `file_url` is required. Python will throw an exception if this parameter isnt' passed.

Run `make docs-clean && make docs-html` to see whether these docstrings are compiled correctly.